app/importers/planning_applications_importer.rb

```diff
@@ -48,12 +52,8 @@ def import_row(row)
      end
    end
 
-   # Not sure how to get the local_authority
-   # Suggestion was from the file name
-   # I would need to know that this was the file to import.
-   # File name could have a date and I could keep a "last imported local_authority: Buckinghamshire, date: 3/9/2023"
    def local_authority
-     @local_authority ||= LocalAuthority.find_or_create_by(name: "Bucks")
+     @local_authority ||= LocalAuthority.find_by!(name: @local_authority_name)
```

Comment:
possibly overkill but I'd be tempted to add a private `attr_reader :local_authority_name` and then make this `name: local_authority_name`

Reply:
Bit more verbose. A bit more consistent usage of access. I'll try it. :nod: